packages/contracts-bedrock/test/L2/OptimismSuperchainERC20.t.sol

```diff
@@ -7,12 +7,14 @@ import { EIP1967Helper } from "test/mocks/EIP1967Helper.sol";
 
 // Libraries
 import { Predeploys } from "src/libraries/Predeploys.sol";
-import { IERC20 } from "@openzeppelin/contracts-v5/token/ERC20/IERC20.sol";
+import { IERC20Solady } from "src/vendor/interfaces/IERC20Solady.sol";
```

Comment:
nit, I would just call this `IERC20` if solc lets us. Using the type named `IERC20Solady` feels like we're leaking implementation details into the type name, and makes this diff noisier
```suggestion
import { IERC20Solady as IERC20 } from "src/vendor/interfaces/IERC20Solady.sol";
```

Reply:
[`83c9701` (#12476)](https://github.com/ethereum-optimism/optimism/pull/12476/commits/83c970116b48f373faa8b247f421eb2d02cdb3ad)